smf/clickhouse/manifest.xml

```diff
@@ -4,7 +4,8 @@
 <service_bundle type='manifest' name='clickhouse'>
 
   <service name='system/illumos/clickhouse' type='service' version='1'>
-  <create_default_instance enabled='false' />
+  <create_default_instance enabled='true' />
+  <single_instance />
 
```

Comment:
`<single_instance/>` is obsolete; see https://www.illumos.org/issues/15477

Reply:
Removed